invenio_app_rdm/theme/views.py

```diff
@@ -33,7 +33,19 @@ def create_blueprint(app):
         static_folder="static",
     )
 
-    blueprint.add_url_rule(routes["index"], view_func=index)
+    # Generate rule from string or tuple
+    def create_url_rule(rule, name, default_view_func):
+        if isinstance(rule, tuple):
+            path, view_func = rule
+
+            if view_func == None:
+                view_func = default_view_func
+
+            return path, name, view_func
+        else:
+            return rule, name, default_view_func
+
+    blueprint.add_url_rule(*create_url_rule(rule=routes["index"], name="index", default_view_func=index))
```

Comment:
I think it makes sense to use the same way of constructing this for the rest of the routes i.e robots, help_search, help_statistics to be consistent...

Reply:
I added the `name` there because I couldn't specify which of the arguments were the `view_func` in `add_url_rule` through the other function (or is there a way?), so it was failing. But I can remove it from the function parameters and just add `None` in the return value?